user-docs/troubleshooting.md

```diff
@@ -0,0 +1,12 @@
++++
+section_weight = 3
```

Comment:
@papr - let me think about the best location for a "Troubleshooting" section - or if troubleshooting tips should just be included within existing sections as as aside. I will discuss this with @mkassner as well